### PR TITLE
Added new symptoms to Liberia followup

### PR DESCRIPTION
**Pull request by obdulia-losantos**

### CODE REVIEW

**Comment by danse** [comment]
this is adding features, so there should be a minor version change. also the change log should be updated

**Comment by danse** [comment]
@rmehner could you please have a look? you wrote the service for the apps and you understand better than me how the customisation logic for the countries works

File: json/lists.json [hunk 1]
```diff
@@ -37,6 +37,6 @@
   "case": []
 }, {
   "countryCode": "lr",
-  "followUp": [],
+  "followUp": ["red_eyes","other_symptoms"],
```
Comment (rmehner) [inline json/lists.json]: This will **only** show those two symptoms for Liberia now.

Comment (obdulia-losantos) [inline json/lists.json]: Should I write the previous **whole** list with these two symptoms?

Comment (rmehner) [inline json/lists.json]: Yep, that is how it works right now. Not really sophisticated, but works for the moment. This is the easiest way, or else we would have to think about how we deal with exclusions and inclusions. The logic is to check for specific symptoms per country and if there are none, simply falling back to the default list.

**Comment by rmehner** [comment]
:+1: looks good to me.

**Pull request merged**